__tests__/app.test.js

```diff
@@ -67,7 +67,37 @@ describe("app!!", ()=>{
           expect(sortedArticlesKeys).toEqual(expect.arrayContaining(['article_id', 'comment_count']))
         })
       })
+      //            TASK 6
+        test('should return comments if there is any',()=>{
+          return request(app)
+          .get('/api/articles/1/comments')
+          .expect(200)
+          .then((response)=>{
+            const comments = response.body.comments
+            //dynamic expected needed (flexible test)
+            //iterate through the input comments with article_id1
+            for(let i =0; i< comments.length; i++){
+              expect(comments[i]).toHaveProperty('article_id', 1)
+            }
```

Comment:
Ideally we'd also check the shape of the comments and that each of them had the correct properties 🔍 
You could do this using toHaveOwnProperty or alternatively toMatchObject